src/main/java/at/hannibal2/skyhanni/features/misc/GhostCounter.kt

```diff
@@ -0,0 +1,326 @@
+    private val ghostlybootPattern = "§6§lRARE DROP! §r§9Ghostly Boots §r§b\\([+](?<mf>.*)% §r§b✯ Magic Find§r§b\\)".toPattern()
+    private val coinsPattern = "§eThe ghost's death materialized §r§61,000,000 coins §r§efrom the mists!".toPattern()
+    private val skillXPPattern = ".*§3\\+(?<gained>.*) .* \\((?<total>.*)\\/(?<current>.*)\\).*".toPattern()
+    private val killComboPattern = "[+]\\d+ Kill Combo [+](?<coin>.*) coins per kill".toPattern()
+    private val killComboExpiredPattern = "§cYour Kill Combo has expired! You reached a (?<combo>.*) Kill Combo!".toPattern()
+    private val scavengerMap = mapOf(
+            1 to 0.3,
+            2 to 0.6,
+            3 to 0.9,
+            4 to 1.2,
+            5 to 1.5,
+    )
+    private var lastXp: String = "0"
+    private var gain: Int = 0
+    private var num: Double = 0.0
+    private var inMist = false
+    private var hasScavengerTalisman = false
+    private val session = Session(
+            0.0,
+            0.0,
+            0.0,
+            0.0,
+            0.0,
+            0.0,
+            0.0,
+            0.0,
+            0.0,
+    )
+
+    val map: MutableMap<Option, Double> = mutableMapOf(
+            KILLS to 0.0,
+            SORROWCOUNT to 0.0,
+            VOLTACOUNT to 0.0,
+            PLASMACOUNT to 0.0,
+            GHOSTLYBOOTS to 0.0,
+            BAGOFCASH to 0.0,
+            TOTALDROPS to 0.0,
+            GHOSTSINCESORROW to 0.0,
+            TOTALMF to 0.0,
+            SCAVENGERCOINS to 0.0,
+            MAXKILLCOMBO to 0.0
+    )
+
+    @SubscribeEvent
+    fun onRenderOverlay(event: GuiRenderEvent.GameOverlayRenderEvent) {
+        if (!isEnabled()) return
+        if (config.onlyOnMist && !inMist) return
+        config.position.renderStringsAndItems(display,
+                extraSpace = config.extraSpace,
+                posLabel = "Ghost Counter")
+    }
+
+    private fun formatDisplay(map: List<List<Any>>): List<List<Any>> {
+        val newList = mutableListOf<List<Any>>()
+        for (index in config.ghostDisplayText) {
+            newList.add(map[index])
+        }
+        return newList
+    }
+
+    fun update() {
+        if (!isEnabled()) return
+        val counter = hidden?.ghostCounter ?: return
+        counter.ghostKills = KILLS.get()
+        counter.sorrowCount = SORROWCOUNT.get()
+        counter.voltaCount = VOLTACOUNT.get()
+        counter.plasmaCount = PLASMACOUNT.get()
+        counter.ghoostlyBootsCount = GHOSTLYBOOTS.get()
+        counter.bagOfCashCount = BAGOFCASH.get()
+        counter.totalDrops = TOTALDROPS.get()
+        counter.ghostSinceSorrow = GHOSTSINCESORROW.get()
+        counter.totalMF = TOTALMF.get()
+        counter.scavengerCoins = SCAVENGERCOINS.get()
+        counter.hasScavengerTalisman = hasScavengerTalisman
+        counter.maxKillCombo = MAXKILLCOMBO.get()
+        display = formatDisplay(drawDisplay())
+    }
+
+    @SubscribeEvent
+    fun onConfigLoad(event: ConfigLoadEvent) {
```

Comment:
You can save the `Map<Option, Double>` directly into the config, then you dont need to use config load